### PR TITLE
Add feedback from 6863

### PR DESCRIPTION
**Pull request by rkoron007**
Addressing @glasser's additional feedback from: https://github.com/apollographql/apollo-server/pull/6863

### CODE REVIEW

File: docs/source/monitoring/health-checks.mdx [hunk 1]
```diff
@@ -21,4 +21,39 @@ Note that this health check will run an actual GraphQL operation. If your server
...
+
+// Requests to `http://localhost:4000/health` now return "Okay!"
+app.get('/health', (req, res) => {
+  res.status(200).send('Okay!');
+});
+```
+
+</MultiCodeBlock>
+
+Note, if you are using `startStandaloneServer`, you must [swap to using the `expressMiddleware` function](../api/standalone/#swapping-to-expressmiddleware) to create an HTTP server health check.
```
Comment (glasser) [inline docs/source/monitoring/health-checks.mdx:59]: comma splice after first word?

File: docs/source/workflow/requests.md [hunk 2]
```diff
@@ -11,7 +11,7 @@ Apollo Server accepts queries and mutations sent as `POST` requests. It also acc
 
 ## POST requests
 
-Apollo Server accepts `POST` requests with a JSON body. A valid request contains a `query` field, along with optional `variables` and an `operationName` (if `query` contains multiple possible operations). You must specify a `Content-Type` HTTP header with type `application/json`.
+Apollo Server accepts `POST` requests with a JSON body. A valid request contains a `query` field, along with optional `variables`, `extensions`, and an `operationName` (if `query` contains multiple possible operations). You must specify a `Content-Type` HTTP header with type `application/json`.
```
Comment (glasser) [inline docs/source/workflow/requests.md:14]:
```suggestion
Apollo Server accepts `POST` requests with a JSON body. A valid request contains a `query` field, along with optional `variables`, `extensions`, and an `operationName` (which disambiguates if `query` contains multiple operations). You must specify a `Content-Type` HTTP header with type `application/json`.
```

I think the previous (and prior to your PR) wording may have inaccurately implied that `operationName` is only legal if `query` contains multiple operations.

**Review by glasser — APPROVED** [review]

File: docs/source/monitoring/health-checks.mdx (hunk 1 above)
Comment (rkoron007) [reply docs/source/monitoring/health-checks.mdx:59]: Good catch. I actually like this better as a note— so I'll remove the comma splice and rephrase like so:
> > If you are using `startStandaloneServer`, you must first [swap to using the `expressMiddleware` function](../api/standalone/#swapping-to-expressmiddleware) before creating an HTTP server health check.

File: docs/source/workflow/requests.md (hunk 2 above)
Comment (rkoron007) [reply docs/source/workflow/requests.md:14]: You raise a good point but I think `disambiguates` feels a little forced here, how does this feel:
> Apollo Server accepts `POST` requests with a JSON body. A valid request contains a `query` field, along with optional `variables`, `extensions`, and an `operationName` (clarifying which `query` to run if the request contains multiple possible operations).

**Comment by rkoron007** [comment]
Going to go ahead and merge this one since you approved @glasser, but please let me know if you'd like to go back on forth on anything else :)

**Comment by glasser** [comment]
lgtm

**Pull request merged**